src/modules/commandrunner/testrecipe.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "RecipeName": "CommandRunner-commandStatus",
+        "ComponentName": "CommandRunner",
+        "ObjectName": "commandStatus",
+        "Desired": false,
+        "ExpectedResult": 0,
+        "WaitSeconds": 0
+    },
+    {
+        "RecipeName": "CommandRunner-InvalidArguments",
```

Comment:
>"RecipeName": "CommandRunner-InvalidArguments", [](http://example.com/codeflow?start=8&length=47)

There is no need for this... These are test object payloads. Recipes are collections of such test object payloads arranged in sequences in JSON files.

Please remove this field from the test objects